src/layers/CasaLayer/CasaLayer.js

```diff
@@ -157,6 +166,42 @@ class CasaLayer extends VectorLayer {
     });
 
     const olStyles = {};
+    if (feature && CasaLayer.isRoute(feature.getGeometry())) {
+      /* Set route start and end style on selected routes */
+      if (isSelected) {
+        const circleStyle = (coords) =>
```

Comment:
Since it's a getter function, call it `getCircleStyle()` and move it out of the styling function. Otherwise the function is newly created on every call of the styling function. That's bad for performance.

Reply:
Good point